### PR TITLE
Makes plasma heal plasmamen

### PR DESCRIPTION
**Pull request by swindly**
:cl: Swindly
add: Plasma now heals plasmamen
/:cl:

[]: # Because plasma poisoning men made out of plasma is bad

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Add an option to either add to the effects of the chem or override them completely so milk will still have the tiny extra healing chance in addition to the bonus heals

**Comment by Cyberboss** [comment]
Uhh, this mean plasmamen will heal from their suits. IIRC breathing plasma injects you with the plasma reagent.

**Comment by PKPenguin321** [comment]
If that were true they would already be getting killed every round by their suits

**Comment by swindly** [comment]
Breathing plasma gas adds the reagent if the lung's safe toxins max is exceeded. Plasmaman lungs have no maximum toxin level.

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm [hunk 1]
```diff
@@ -67,4 +67,14 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 	if(lastname)
 		randname += " [lastname]"
 
-	return randname
\ No newline at end of file
+	return randname
+	
+/datum/species/plasmaman/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(chem.id == "milk")
+		H.heal_bodypart_damage(1, 0, 0)
```
Comment (PKPenguin321) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm]: thatsthejoke.jpg

**Comment by Lexorion** [comment]
What if someone tries to get a human lung to abuse the healing? That could work,right?

**Comment by optimumtact** [comment]
Why should it heal them as opposed to just being neutral?

Humans don't heal in an airmix do we?

File: code/modules/mob/living/carbon/human/species_types/plasmamen.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mob/living/carbon/human/species_types/plasmamen.dm]: please don't post long images, link to them

**Comment by optimumtact** [comment]
Also add the milk change to your op please

**Comment by swindly** [comment]
Milk has been able to heal plasmamen since https://github.com/tgstation/tgstation/pull/16206

**Comment by optimumtact** [comment]
So wait, what did you change to it?

**Comment by optimumtact** [comment]
Oh I see you refactored the mob specific changes to live on those mobs

Yeah that's probably the right place for them to live

**Comment by PKPenguin321** [comment]
Refactored it to it's own thing in reagent process or whatever instead of a race check in milk itself

**Comment by SweaterKittens** [comment]
As the "maintainer" of Plasmamen, I'm not sure how I feel about them being healed by plasma. They absolutely should not be getting poisoned by it though, that's for sure. 

I don't think it's much of a balance concern, as liquid plasma is hard enough to come by, and would require a decent amount of effort to get in syringes or a hypo. Honestly, my only real concern is that I'm not sure it really make sense for it to heal them when they're made of the stuff. And if it did heal them, why wouldn't it heal them while they were constantly inhaling it?

**Comment by XDTM** [comment]
To prevent lung abuse it should simply remove the reagent with no effect.

**Comment by swindly** [comment]
>lung abuse
Plasmamen won't even get cloned. Good luck getting someone to do surgery on you.

If you do get human lungs, you'll suffocate if you use plasma tanks as internals because you'll need oxygen instead of plasma.

**Comment by PKPenguin321** [comment]
>To prevent lung abuse it should simply remove the reagent with no effect.

Or keep the healing but check for proper lungs instead of just mutantrace

**Comment by InsaneHyena** [comment]
All alien races should be worse than humans.

**Comment by swindly** [comment]
Anyone who thinks this will make plasmamen better than humans has never played as a plasmaman.

**Comment by optimumtact** [comment]
@tgstation/commit-access I'm not sold on this, so if no-one else merges within two days I will close it, as i think it's just an unnecessary buff to plasmemes and also doesn't make much ic sense.

**Comment by RemieRichards** [comment]
plasma + plasma = 2plasma
makes perfect sense.

**Comment by KorPhaeron** [comment]
I don't really care

**Comment by RemieRichards** [comment]
the most "meh" PR ever? <i>let's find out!</i>

**Comment by optimumtact** [comment]
I guess the other maintainers didn't care and I don't like it because it's a buff with no real gameplay reasoning.

**Pull request closed without merging**